src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResource.kt

```diff
@@ -572,3 +579,29 @@ data class CourtAppearanceRequest(
   val judgeName: String?,
    */
 )
+
+@Schema(description = "Court Event")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class OffenderChargeRequest(
+  val offenceCode: String,
+  val statuteCode: String,
+  val offencesCount: Int?,
+  val offenceDate: LocalDate?,
+  val offenceEndDate: LocalDate?,
+  val resultCode1: String?,
+  val mostSeriousFlag: Boolean,
+
+  /*
+  val plea: String?,
+  val propertyValue: BigDecimal?,
+  val totalPropertyValue: BigDecimal?,
+  val cjitCode1: String?,
+  val cjitCode2: String?,
+  val cjitCode3: String?,
+  val chargeStatus: String?,
+  val resultCode2: String?,
+  val resultCode1Indicator: String?,
+  val resultCode2Indicator: String?,
+  val lidsOffenceNumber: Int?,
+  */
```

Comment:
I'm leaving these comments as a reminder that further discussion of these fields is required